x-pack/plugins/maps/public/classes/sources/es_search_source/util/scaling_form.tsx

```diff
@@ -90,7 +90,7 @@ export class ScalingForm extends Component<Props, State> {
       <EuiRadio
         id={SCALING_TYPES.CLUSTERS}
         label={i18n.translate('xpack.maps.source.esSearch.clusterScalingLabel', {
-          defaultMessage: 'Show clusters when results exceed {maxResultWindow}.',
+          defaultMessage: 'Show clusters when results exceed {maxResultWindow}',
```

Comment:
@gchaps is it ok to remove the periods from these. They are used in the UI like this

<img width="300" alt="Screen Shot 2021-07-12 at 8 12 16 AM" src="https://user-images.githubusercontent.com/373691/125302196-f5428700-e2e8-11eb-9a52-62a964219d0a.png">

<img width="300" alt="Screen Shot 2021-07-12 at 8 12 42 AM" src="https://user-images.githubusercontent.com/373691/125302219-fa9fd180-e2e8-11eb-98b7-5ea7f2cf6113.png">

Reply:
Yes, remove the ending period for radio buttons

- Limit results to 10000
- Show clusters when results exceed 100000
- Use vector tiles